src/Core/Formatter.cpp

```diff
@@ -22,14 +22,14 @@
 
 namespace Core
 {
-Formatter::Formatter(QString clangFormatBinary, const QString &clangFormatStyle, MessageLogger *log)
+Formatter::Formatter(const QString &clangFormatBinary, const QString &clangFormatStyle, MessageLogger *log)
 {
     this->log = log;
-    updateBinary(clangFormatBinary);
-    updateStyle(clangFormatStyle);
+    Formatter::updateBinary(clangFormatBinary);
```

Comment:
Do not fully qualify the name. It may confuse users to think that updateBinary is a static function.